calculator.go

```diff
@@ -1,13 +1,127 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+)
+
+// Add takes a variable number of arguments and returns the
+// result of adding them together.
+func Add(inputs ...float64) float64 {
```

Comment:
Great! But one potential problem with this is that we could quite feasibly pass _no_ parameters at all, or only one parameter, neither of which really makes sense. We could check the length of `inputs` inside the function, but that only catches the problem at run-time. Is there a way we could declare this function so as to make it a _compile_ error if the user doesn't pass two or more parameters?

Reply:
Definitely slapped my forehead a bit when I realized my code wasn't enforcing some basic arithmetic rules!